src/emu/softlist.cpp

```diff
@@ -125,8 +122,8 @@ const char *software_part::feature(const char *feature_name) const
 
 	// scan the feature list for an entry matching feature_name and return the value
 	for (const feature_list_item &feature : m_featurelist)
-		if (strcmp(feature.name(), feature_name) == 0)
-			return feature.value();
+		if (strcmp(feature.name().c_str(), feature_name) == 0)
```

Comment:
if you're doing a literal comparison, you may as well just use `operator==` when you have a `std::string`, there's no need for a roundabout `strcmp`